templates/decision/record.tpl

```diff
@@ -52,89 +52,81 @@
 				:id="step.id"
 				:label="step.name"
 			>
-				<template v-if="step.type === 'form'">
-					<panel>
+				<panel class="decision__stepPanel">
+					<panel-section class="decision__stepHeader">
+						<h2>{{ step.name }}</h2>
+						<p>{{ step.description }}</p>
+					</panel-section>
+					<template v-if="step.type === 'form'">
 						<panel-section>
-							<template slot="header">
-								<h2>{{ step.name }}</h2>
-								<p>{{ step.description }}</p>
-							</template>
 							<pkp-form v-bind="step.form" @set="updateStep"></pkp-form>
 						</panel-section>
-					</panel>
-				</template>
-				<template v-else-if="step.type === 'email'">
-					<panel v-if="skippedSteps.includes(step.id)">
-						<panel-section>
-							<template slot="header">
-								<h2>{{ step.name }}</h2>
-								<p>{translate key="editor.decision.emailSkipped"}</p>
-							</template>
+					</template>
+					<template v-else-if="step.type === 'email'">
+						<panel-section v-if="skippedSteps.includes(step.id)">
+							<notification type="warning">
+								{translate key="editor.decision.emailSkipped"}
```

Comment:
Just a generic note unrelated to this PR... As this will be processed by Vue after Smarty, a translation with `{{x}}` could break something and, already out of the Vue context, if we're not reviewing translations carefully, someone could attempt to make a "translation injection".